extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -587,7 +586,9 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	getNavigation(uri: vscode.Uri): Thenable<azdata.nb.NavigationResult> {
 		let result: azdata.nb.NavigationResult;
 		let notebook = this.currentBook?.getNotebook(uri.fsPath);
-		if (notebook) {
+		// checking viewId to verify that the book is opened in the Notebooks Viewlet
+		// which indicates if navigation buttons should be visible or hidden.
+		if (this.viewId === constants.BOOKS_VIEWID && notebook) {
```

Comment:
What is this fixing? Don't we still want navigation buttons in the provided books too?

Reply:
If the book is closed, we don't want to show the navigation buttons in the pinned notebooks. I'll modify the condition because it's not working for Provided Books.